components/brave_ads/core/internal/creatives/creative_ads_database_table.cc

```diff
@@ -38,6 +38,7 @@ void BindRecords(mojom::DBCommandInfo* command) {
   command->record_bindings = {
       mojom::DBCommandInfo::RecordBindingType::
           STRING_TYPE,  // creative_instance_id
+      mojom::DBCommandInfo::RecordBindingType::STRING_TYPE,  // creative_set_id
```

Comment:
Do we need a new column for a performance fix?

Reply:
We do because the issue was that when purging ad events, we need to know if a creative ad is still active, which requires the creative_set_id

Reply:
hmm, I don't see that new column is used with ad_events purge query. maybe I missed something

Reply:
See `AdEvents::PurgeExpired` and `WHERE creative_set_id NOT IN (SELECT creative_set_id from creative_ads)`

Reply:
I see, so we queried non-existing column and now it is fixed.

Reply:
That is correct, so basically the ad_events table would keep growing in size without ever being pruned.